rclcpp_components/cmake/rclcpp_components_add_node.cmake

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create a target which builds a shared library and also creates a target
+# which builds an executable
+# :param EXCLUDE_FROM_ALL: exclude the target from default build target 
+# :type EXCLUDE_FROM_ALL: boolean
+# :param ARGS_TARGET: the name of the shared library 
+# :type ARGS_TARGET
```

Comment:
I guess the type is "string". Also, following my previous comment, I would rename this param to `target` and `ARG_SRC` below to `sourceN`.